### PR TITLE
Admin page

### LINKED ISSUE #14: Admin page
We need an admin page that will link to the different CRUD actions admins can take. This page will need some logic to determine what kind of access the user has.

Story points: 3

### PR DESCRIPTION
**Pull request by subnomo**
Fixes #14 
## Changes in this PR.

- [X] Add admin page that links to CRUD pages
## Testing this PR.

1. Run `docker-compose build`
2. Run `docker-compose up`
3. Navigate to http://0.0.0.0:5000/admin, logging in if necessary
### Expected Output.

If you're not logged in as admin, you should get `You cannot access this page.`
If you _are_ logged in as admin, you should see this:

@osuosl/devs

### CODE REVIEW

**Comment by aaroncohen73** [comment]
Currently, if getUser() returns an error of any sort (Such as a failure to connect to the server), the index page doesn't appear and is replaced with an error 500 page. Instead of this, the index page should appear as it would for a user who is not logged in.

**Comment by subnomo** [comment]
@aaroncohen73 I believe this is a separate issue. See #12.

File: app/views.py [hunk 1]
```diff
@@ -39,6 +39,11 @@ def getUser():
                            token=session['token'])
     user = ts.get_users(username=session['username'])
 
+    if (type(user) is list and session['username'] == 'admin' and
```
Comment (matthewrsj) [inline app/views.py]: Can you comment this? Why are you checking if `type(user) is list`?

**Comment by matthewrsj** [comment]
One small section I'm confused about that needs a comment. +1 once that is fixed.

**Comment by subnomo** [comment]
@matthewrsj Good catch, fixed. I was checking if `user` was a list because the current version of pymesync returns a dict for single users, while test mode returns a list. Then I remembered that I can just use `app.config['TESTING']`, and forgot to remove the list check.

**Comment by matthewrsj** [comment]
Thanks @subnomo, +1

**Comment by aaroncohen73** [comment]
+1

**Pull request merged**